treefmt.toml

```diff
@@ -6,3 +6,7 @@ includes = [ "*.nix" ]
 command = "rustfmt"
 options = ["--edition", "2021"]
 includes = [ "*.rs" ]
+
+[formatter.markdown]
```

Comment:
A small thing, but could the formatters be kept in alphabetical order by filetype.